test/multiverse/suites/padrino/config/newrelic.yml

```diff
@@ -7,6 +7,8 @@ development:
   monitor_mode: true
   license_key: bootstrap_newrelic_admin_license_key_000
   ca_bundle_path: ../../../config/test.cert.crt
+  instrumentation:
+    sinatra: <%= $instrumentation_method %>
```

Comment:
is "sinatra" correct here?  Or should it be "padrino"?

Reply:
Yeah, our padrino instrumentation previously relied on the sinatra configs and didn't have it's own config option. They need to be in sync with each other for prepend vs chain too, since padrino uses some of the sinatra instrumentation. In the padrino instrumentation, I added `configure_with :sinatra` so that it would be looking at sinatra config value.

Reply:
perfect!